apps/dotcom/spaRouteFilter.ts

```diff
@@ -0,0 +1,19 @@
+// This value is used to configure Vercel routing to rewrite dotcom SPA routes to /index.html.
+// It is also tested in routes.test.ts to make sure it matches all React Router routes.
+//
+// It is a string and not a regular expression as it's substituted
+// into a string in Vercel's "build output spec" in scripts/build.ts.
+//
+// Make sure it's not overly broad, because otherwise we won't give correct 404 responses.
+export const SPA_ROUTE_FILTER =
+	'^\\/new.*' +
+	'|' +
+	'^\\/r' +
+	'|' +
+	'^\\/r\\/.*' +
+	'|' +
+	'^\\/s\\/.*' +
+	'|' +
+	'^\\/v\\/.*' +
+	'|' +
+	'^\\/$'
```

Comment:
nit: this might be easier to read if you use the backtick string templates?
or, actually, could this be simplified to be: `^/new.*|^/r|^/[rsv]/.*|^/$`

actually, aren't these equivalent: 
'^\\/r'
and 
'^\\/r\\/.*'

Reply:
small bug: ^/new.* would capture /newblahblah which shouldn't be captured. should be: ^/new$

Reply:
good points!

re: `\\/`: you're absolutely right, I I was experimenting with the standard regex syntax and I didn't realise I don't need to escape slashes in strings. My bad 😅 

I also simplified the regex in a different way, please check out the last commit!